driver/src/main/java/oracle/nosql/driver/NoSQLHandleConfig.java

```diff
@@ -581,8 +581,8 @@ public NoSQLHandleConfig setRequestTimeout(int timeout) {
     }
 
     /**
-     * Sets the default table request timeout. The default timeout is
-     * 5 seconds. The table request timeout can be specified independently
+     * Sets the default table request timeout.
+     * The table request timeout can be specified independently
      * of that specified by {@link #setRequestTimeout} because table requests can
      * take longer and justify longer timeouts. The default timeout is 10
      * seconds (10000 milliseconds).
```

Comment:
Am I reading this diff right? Shouldn't we be taking out the line about the default timeout being 10 seconds -- isn't it 5 seconds?

Reply:
Or is it 5 for regular requests and 10 seconds for table requests?

Reply:
Correct -- 5 for regular requests and that is documented. 10 for table requests.